.eslintrc.json

```diff
@@ -231,11 +231,33 @@
         "*.json5"
       ],
       "extends": [
-        "plugin:jsonc/recommended-with-jsonc"
+        "plugin:jsonc/recommended-with-json5"
       ],
       "rules": {
-        "no-irregular-whitespace": "error",
+        // ESLint core no-irregular-whitespace rule doesn't work well in JSON
+        "no-irregular-whitespace": "off",
```

Comment:
I think you might have misunderstood what this rule does, this does not verify if the correct amount of spaces were used (like `jsonc/no-irregular-whitespace`), but it checks if any [irregular whitespaces](https://eslint.org/docs/latest/rules/no-irregular-whitespace#rule-details) were used. An example of one of those irregular whitespaces was for example on line 1061 in `fr.json5` in [this commit](https://github.com/DSpace/dspace-angular/commit/3c2f3d071fdca751202809a75dc4dfbc3a030699). When I ran that command in the `fr.json5` it removed 27 No-Break Spaces. These spaces prevent the text to wrap with `text-wrap` so that's why I enabled it

Reply:
Thanks @alexandrevryghem, it's been a while since I looked at this PR. Looking at it now, I see this commit disables eslint's core `no-irregular-whitespace` plugin and uses `jsonc/no-irregular-whitespace` instead, as recommended by the jsonc plugin. The new plugin is set to `error` just like it was previously, but uses some overrides—maybe I can disable to keep everything the same.

Also, I don't see any non-breaking spaces (U+00A0) in `fr.json5` any more on the `main` branch? [Your commit](https://github.com/DSpace/dspace-angular/commit/3c2f3d071fdca751202809a75dc4dfbc3a030699) remove actually removed them.